pycqed/measurement/calibration/two_qubit_gates.py

```diff
@@ -1379,7 +1414,6 @@ def sweep_block(self, sweep_points,
         w.block_end.update({'pulse_delay': max_flux_length * num_cz_gates})
         fp_w = self.simultaneous_blocks('sim', [fp, w], block_align='center')
 
-        self.data_to_fit.update({qbr: 'pe'})
```

Comment:
What happened to `data_to_fit`?

Reply:
The default in MultiQubit_TimeDomain_Analysis is fine for Chevron measurements.